freqtrade/rpc/fiat_convert.py

```diff
@@ -127,6 +136,9 @@ def _find_price(self, crypto_symbol: str, fiat_symbol: str) -> float:
         if crypto_symbol == fiat_symbol:
             return 1.0
 
+        if self._cryptomap == {} and self._backoff <= datetime.datetime.now().timestamp():
+            self._load_cryptomap()
```

Comment:
I'm not sure about the impact of this ... 

it's called from (semi)random spots in the code - so doing 2 calls "inline" might slow down certain points - and if that again runs into the ratelimit - it might not fix anything after all ...

Reply:
just to be clear - i'm not against this in general, and i guess it might help some people.

A warning when this happens would for sure be helpful - so it's obvious that this happened.
You might need to handle the case of "reloading fails too" differently though - as now, 2 reloads would result in "unsupported crypto-symbol" - which is kind of wrong.

Reply:
This is now handled below by returning 0.0 if _cryptomap is still `{}`